templates/display_product.tt.html

```diff
@@ -52,7 +52,8 @@ <h1 property="food:name" itemprop="name">[% title %]</h1>
 <!-- Display UPC code if the EAN starts with 0 -->
 [% IF upc_code == 'defined' %]
     <p id="barcode_paragraph">
-        [% lang("barcode") %]:  <span id="barcode" property="food:code" itemprop="gtin13" style="speak-as:digits;">[% code %]</span>[% upc %]
+        [% lang("barcode") %]: <p id="_barcode">[% code %]</p>
```

Comment:
Doesn't this nest two `p` elements? I don't think that's a recommendation HTML structure. What's bad about the previous `span`?

Reply:
That p tag is removed by the above javascript code, i actually used it to get the barcode text inside the javascript code i.e. innerHTML